index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width">
+        <meta name="author" content="Amos Gloria">
+        <title>Facebook Clone | Welcome</title>
+        <link rel="stylesheet" href="./css/style.css">
+        </head>
+        <body>
+            <header>
+                <div class="header-block">
+                    <div id="branding">
+                        <h1><span class="highlight">facebook</span></h1>
+                        </div>
+                        <nav>
+                            <div class="facebool-lpgin">
+                                <div class="email">
+                                <label>Email or Phone</label><br>
+                                <input type="email or phone" placeholder="Email"><br>
+                                <label>Keep me logged in</label>
```

Comment:
You omitted the checkbox to keep you logged in. Review the given design to see it